modules/administration-guide/partials/proc_defining-the-list-of-images-to-pull.adoc

```diff
@@ -1,6 +1,14 @@
 [id="defining-the-list-of-images-to-pull_{context}"]
 = Defining the list of images to pull
 
+The {image-puller-name-short} can pre-pull most images, including scratch images such as `che-machine-exec`. However, images that mount volumes in the Dockerfile, such as `traefik`, are not supported for pre-pulling on OpenShift and are not a good use case for speeding up the workspace start.
+
+Pre-pulling images, which are involved during workspace startup, leads to speeding up the workspace start times. For example:
```

Comment:
`Pre-pulling images involved in workspace startup will reduce workspace start times. For example:`